oat-sa/tao-core
# Feat/aut 3675/add support to solar UI locales

### Related to: https://oat-sa.atlassian.net/browse/AUT-3675

### Installation
Need to use tao-community with the June release
Then enable Solar theme [https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/2406350936/RFE-1147+Implementation+documentation](https://oat-sa.atlassian.net/wiki/spaces/OAT/pages/2406350936/RFE-1147+Implementation+documentation)

### What to test

- Check Solar theme enabled
- try to run commands:
```
php tao/scripts/taoTranslate.php -v -a=compile -e=tao -l=en-US-S
php tao/scripts/taoUpdate.php
```


### Summary

It makes translations with new prefix
Business logic

To create and update new local files:
```sh
// from Docker container

php  tao/scripts/taoTranslate.php -v -a=create -e=taoQtiItem -l=en-us```
